stratos/assetlist.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "stratos",
+  "assets": [
+    {
+      "description": "STOS coin is the token for the Stratos (STOS) cosmos based blockchain",
+      "denom_units": [
+        {
+          "denom": "wei",
```

Comment:
I think you might have the denoms backwards. I assume wei should be exp 0, and stos be exp 18 (similar to Wei and ETH))

Reply:
Fixed and also fixed in `fee_tokens` place